packages/ignite/src/templates/plugin/commands/example.js.ejs

```diff
@@ -0,0 +1,26 @@
+// @cliDescription  Example <%= props.name %> command
+// @cliExample  ignite generate example <%= props.name %>
+// Feel free to remove this if you have no need to generate files
+// with your plugin.
+// ----------------------------------------------------------------------------
+module.exports = async function (context) {
+  const { parameters, strings, print, ignite } = context
+  const { pascalCase, isBlank } = strings
+
+  // validation
+  if (isBlank(parameters.first)) {
+    print.info(`ignite generate example <name>\n`)
+    print.info('A name is required.')
+    return
+  }
+
+  const name = pascalCase(parameters.first)
+
+  const copyJobs = [{
+    template: 'example.ejs',
+    target: `App/Examples/${name}.js`
```

Comment:
This isn't actually an Ignite example like you guys are thinking. I sort of poorly named this file. This is a template file that is generated along with the plugin, intended for modification.

Replace "Examples" with "Components" to get a sense for what we're doing in this file. You could create a Component generator with this file. The "Examples" folder is just a placeholder that will be renamed by the plugin author.